src/components/InlineCodeBlock/WrappedText.tsx

```diff
@@ -40,15 +42,46 @@ function containsEmoji(text: string): boolean {
     return CONST.REGEX.EMOJIS.test(text);
 }
 
+/**
+ * Splits long words into multiple strings
```

Comment:
Can we add more to the comment here, explaining why it is that we have to do this.

Reply:
Of course, I've added a more elaborate explanation for it.